src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArray = sourceString.split(';');
+  const styleObject = {};
+
+  styleArray.forEach((style) => {
+    if (style.trim()) {
+      const [key, ...valueParts] = style.split(':');
```

Comment:
The use of array destructuring here is correct, but ensure that `key` is not an empty string after trimming. This will help avoid adding empty keys to the `styleObject`.